### ISSUE TITLE
Prepare assets for Kushal to run a "real" 'Promote a Blog Post' campaign.

### ISSUE THREAD

**Issue by paagla123** [coa0329/BhaangWeb#100]
_From @daver78 on September 13, 2017 13:20_

@Mcfarsheed, as discussed, I want you to have Kushal run one of your "Tuesday" content promotions.  He'll use the Blueprint to run it and thru that process, discover additional areas of polish that are required.

If he feels he needs another Campaign or two to continue improving things, we'll open up another issue.

If not, we'll then ask you to run a Campaign and provide @paagla123 with feedback.

Basically, here's what @paagla123 needs from you:

- URL to a blog post you want to promote
- The UTM string you want to use
- The email content you want to use, including hero image at top
- When you want emails 2 and 3 to hit 

I think that's it, but @paagla123 can add to this if there are other things he needs.

_Copied from original issue: coa0329/Bhaang#108_

**Comment by paagla123** [comment]
@daver78 we need to get the MindFire logo back...
If this is for real we need to add daVinci to MindFire Master

**Comment by paagla123** [comment]
_From @daver78 on September 15, 2017 3:37_

@paagla123 I think you created an issue for allowing the User to upload their company logo.  Should resolve this, yes?

**Comment by paagla123** [comment]
@Mcfarsheed what's the plan here?

**Comment by paagla123** [comment]
_From @Mcfarsheed on September 26, 2017 22:30_

we were supposed to do it today but I dropped the ball on coordinating with
you. Can we do it next Tuesday? I have mostly everything ready, but just
got held up with trade show stuff.

I'll f/u with you directly to coordinate for next Tues.

Thanks!

Sincerely,

Mackenzi
MindFireInc.
949-842-4388
www.mindfireinc.com

**Comment by paagla123** [comment]
@Mcfarsheed let's chat tomorrow

**Comment by paagla123** [comment]
_From @Mcfarsheed on September 27, 2017 1:58_

ok! I'll call you!

Sincerely,

Mackenzi
MindFireInc.
949-842-4388
www.mindfireinc.com

**Comment by paagla123** [comment]
For you @Mcfarsheed 
https://youtu.be/-mVMvfqnUWs?t=45s

**Comment by paagla123** [comment]
_From @Mcfarsheed on September 27, 2017 15:35_

haha Kush!

Sincerely,

Mackenzi
MindFireInc.
949-842-4388
www.mindfireinc.com

**Comment by daver78** [comment]
@paagla123 Looks like Mac isn't invited to participate in this repo.  @coa0329 Can you invite her?

**Comment by paagla123** [comment]
@coa0329 can you please add mcfarsheed as well as Khalessi@mindfireinc.com to the BhangWeb repo? Thx

**Comment by Mcfarsheed** [comment]
Thanks -- also Kush and I have discussed what he needs to run the real
post, he'll do a podcast/blog post this Thurs and I'll provide him
everything by tomorrow!

Sincerely,

Mackenzi
MindFireInc.
949-842-4388
www.mindfireinc.com

**Comment by coa0329** [comment]
I send invite to @mcfarsheed.

What is github if for Khalessi@mindfireinc.com ?

**Comment by paagla123** [comment]
@coa0329 thank you. I will find out re Kathy.

**Comment by Mcfarsheed** [comment]
I accepted!

Sincerely,

Mackenzi
MindFireInc.
949-842-4388
www.mindfireinc.com

**Comment by paagla123** [comment]
Done!
